src/pages/index.vue

```diff
@@ -1004,11 +1003,23 @@ export default {
         answer,
       }));
     },
+    randomKeywords() {
+      if (this.$i18n.locale === 'zh-Hant') {
+        const shuffled = [...SEARCH_SUGGESTIONS].sort(
+          () => Math.random() - 0.5
+        );
+        const randomTerms = shuffled.slice(0, 3);
+        return randomTerms;
+      }
+      return [];
+    },
   },
   mounted() {
     logRetailEvent(this, 'home-page-view');
     window.addEventListener('scroll', this.handleScroll);
-    this.placeholderText = this.getRandomPlaceholder();
+    this.placeholderText = this.randomKeywords?.length
+      ? this.randomKeywords.join('、')
+      : this.$t('gutenberg_search_placeholder');
```

Comment:
this can be computed

Reply:
[♻️ Refactor placeholderText into a computed prop](https://github.com/likecoin/liker-land/pull/2003/commits/69a6555e5af3f7f24d8e847f380660e341ff84e0)